packages/components/src/popover/src/Popover.tsx

```diff
@@ -118,6 +118,10 @@ export function InnerPopover({
         }
     }), [popoverId]));
 
+    if (isNil(heading) && isNil(ariaLabel)) {
+        console.error("A popover must either have a heading or an \"aria-label\" attribute.");
```

Comment:
do we want console.error or should it be console.warn?

Reply:
In orbiter, it seems to be console.error

Reply:
really? alright then.